ansible_collections/arista/avd/roles/eos_designs/docs/tables/cloudvision-tags.md

```diff
@@ -100,9 +100,126 @@
     # If you need to change all the existing `node_type_keys`, it is recommended to copy the defaults and modify them.
     # If you need to add custom `node_type_keys`, create them under `custom_node_type_keys` - if named identically to default `node_type_keys` entries,
     # custom entries will replace the equivalent default entry.
-    node_type_keys:
+    node_type_keys: # (1)!
       - key: <str; required; unique>
 
         # Device type that CloudVision should use when generating the Topology like "leaf", "spine", "core", "edge" or "member-leaf".
         cv_tags_topology_type: <str>
     ```
+
+    1. Default Value
+
+        ```yaml
+        node_type_keys:
+        - cv_tags_topology_type: spine
```

Comment:
I think it would feel more natural if the `key` / `type` is first when looking for the types